src/common/components/ui/WizardNavItem.tsx

```diff
@@ -22,8 +21,21 @@ const NavItem: React.FC<WizardNavItemProps & { isValid?: () => boolean }> = ({
       </>
     );
   }
+  return content;
+};
+
+const NavItem: React.FC<PropsWithChildren<WizardNavItemProps & { isValid?: () => boolean }>> = ({
```

Comment:
I think the recommendation is to remove `React.FC` and to use  `PropsWithChildren` instead as the type for the function arguments.
Something like:

```
const NavItem = ({
  isValid = () => true,
  children,
  ...props
}: PropsWithChildren<WizardNavItemProps & { isValid?: () => boolean }> ) => {
  const { content, isDisabled, isCurrent } = props;

  return (
    <WizardNavItem {...props} content={getNavItemContent(content, isValid, isDisabled, isCurrent)}>
      {children}
    </WizardNavItem>
  );
};
```